searx/engines/__init__.py

```diff
@@ -57,6 +41,8 @@
 
 
 def load_engine(engine_data):
+    # pylint: disable=too-many-branches, too-many-statements, protected-access
```

Comment:
Do you like to add a doc-string that documents 

- engine attributes from YAML `settings:engines/<engine>` argument `engine_data`
- set default attributes (from `engine_default_args`)
- check _inactive_  is True or False
- what does _"checking required variables"_ means